FirebasePerformance/Sources/FPRClient.m

```diff
@@ -295,7 +295,7 @@ + (void)cleanupClearcutCacheDirectory {
       [[NSFileManager defaultManager] removeItemAtPath:logDirectoryPath error:&directoryError];
 
       if (directoryError) {
-        FPRLogDebug(kFPRClientTempDirectory,
+        FPRLogWarning(kFPRClientTempDirectory,
```

Comment:
For the developers this does not mean anything. This temporary directory was managed by us and this does not cause any impact for the developer apps. So I would leave this as Debug. Definitely not a warning for the developer.

I would recommend picking the log level on what the developer would perceive would be ideal when compared to what the perf SDK would perceive.

Reply:
What is the use of the directory? If it does not have any impact on collecting perf events for the app, what is it used for?

Reply:
This was a directory that was required when dispatching events using clearcut. This directory has to be removed since we are now on Firelog.

This is essentially required for people who are still on the clearcut version of our SDK moving to FIrelog version of our SDK.

Reply:
In that case, this is dead code in the current version of our SDK, right?

Reply:
Discussed offline. Will log with `Debug`. This is likely a dead code and won't happen on the current version of the SDK. I can create another PR to remove related logics.